### PR TITLE
Exclude doc/cprover-manual from Doxygen input [blocks: #3258]

### PR DESCRIPTION
**Pull request by johnnonweiler**
The documentation in the folder doc/cprover-manual was not intended to be included in the Doxygen documentation.  It is intended to be the source for a separate website.

- [X] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [X] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: src/doxyfile [hunk 1]
```diff
@@ -813,7 +813,7 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which doxygen is
 # run.
 
-EXCLUDE                = solvers/z3/
+EXCLUDE                = solvers/z3/ ../doc/cprover-manual
```
Comment (chrisr-diffblue) [inline src/doxyfile]: I'd personally follow the pattern used for other list variables in this file, and separate the two paths by a backslash and new line pair. I think that makes it more readable.

Comment (johnnonweiler) [reply src/doxyfile]: 👍

**Review by kroening — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:warning:
Diffblue compatibility check is currently unavailable.
Please create manual bump.
(cbmc commit: 59c4e6ed0eaa0abec5349fc81bca92b1b16648fc).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/90723201

**Review by peterschrammel — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:warning:
Diffblue compatibility check is currently unavailable.
Please create manual bump.
(cbmc commit: 9b9f2beca46fbf2ab6f48194be1b1013000a5e87).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/90724541

**Pull request merged**